src/ucs/datastruct/queue.h

```diff
@@ -20,6 +20,7 @@
  */
 static inline void ucs_queue_head_init(ucs_queue_head_t *queue)
 {
+    queue->head  = NULL;
```

Comment:
`#ifdef cppcheck` ?

Reply:
sorry, I was wrong this is CLANG warning
to fix NULL subtraction, changed it to
```
queue->head  = (ucs_queue_head_t*)(void*)queue;
```
move it under `#ifdef __clang_analyzer__ ... #endif`

@yosefe 
btw, may be better to not suppress it and always have it initialized? wdyt?
because it can be initialized by NULL